environment.yml

```diff
@@ -7,7 +7,6 @@ dependencies:
   - jupyterlab>=0.35
   - six>=1.11
   - numpy>=1.12
-  - pydot>=1.3.0
```

Comment:
You should instead replace with `graphviz` because we definitely want to have it available e.g. in Binder launches of the demo notebook.